apps/products-feed/src/modules/app-configuration/app-config.ts

```diff
@@ -1,6 +1,14 @@
 import { createLogger } from "@saleor/apps-shared";
 import { z } from "zod";
 
+const imageSizeFieldSchema = z.coerce.number().gte(256).default(1024);
```

Comment:
FYI: `coerce` is needed to fix issues with client-side validation. Without it input fields are rejected as values are submitted as strings